<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyDownlodloader</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #2E2A27;   /* بني غامق */
    color: #FFA500;              /* برتقالي */
    font-family: Arial;
    font-size: 14px;
}

/* Tabs */
QTabWidget::pane {
    border: 1px solid #A0522D;
    background: #2E2A27;
}
QTabBar::tab {
    background: #3B322C;
    color: #FFA500;
    padding: 6px 12px;
    border: 1px solid #A0522D;
    border-top-left-radius: 6px;
    border-top-right-radius: 6px;
}
QTabBar::tab:selected {
    background: #FFA500;
    color: #2E2A27;
}

/* Menu bar */
QMenuBar {
    background-color: #3B322C;
    color: #FFA500;
}
QMenuBar::item:selected {
    background: #FFA500;
    color: #2E2A27;
}
QMenu {
    background-color: #2E2A27;
    color: #FFA500;
    border: 1px solid #A0522D;
}
QMenu::item:selected {
    background: #FFA500;
    color: #2E2A27;
}

/* Labels */
QLabel {
    color: #FFA500;
    font-weight: bold;
}

/* LineEdit */
QLineEdit {
    background-color: #3B322C;
    color: #FFA500;
    border: 1px solid #FFA500;
    border-radius: 4px;
    padding: 4px;
}

/* Buttons */
QPushButton {
    background-color: #FFA500;
    color: #2E2A27;
    border: 2px solid #A0522D;
    border-radius: 6px;
    padding: 6px;
}
QPushButton:hover {
    background-color: #FFB733;
}
QPushButton:pressed {
    background-color: #CC8400;
}</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>791</width>
     <height>491</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Downlode Youtube Video</string>
    </attribute>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>350</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Start Downlod</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>URL</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>290</y>
       <width>641</width>
       <height>41</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>659</x>
       <y>120</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>221</x>
       <y>120</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>49</y>
       <width>511</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>66</x>
       <y>132</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Save Location</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>205</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Qualty</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>200</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1080p</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>720p</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>360p</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>144p</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Downlode Youtube Playlist</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>221</x>
       <y>49</y>
       <width>511</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>659</x>
       <y>120</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>74</x>
       <y>60</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>URL</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>132</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Save Location</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>200</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1080p</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>720p</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>360p</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>144p</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>221</x>
       <y>120</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding : 0px ;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>74</x>
       <y>205</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Qualty</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>340</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Start Downlod</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>280</y>
       <width>641</width>
       <height>41</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
